tests/models/test_anthropic.py

```diff
@@ -2163,3 +2163,80 @@ async def test_anthropic_web_search_tool_stream(allow_model_requests: None, anth
 
 Additional notable stories include Vietnam's plan to ban fossil-fuel motorcycles in the heart of Hanoi starting July 2026, aiming to cut air pollution and move toward cleaner transport, and ongoing restoration efforts for Copenhagen's Old Stock Exchange, which is taking shape 15 months after a fire destroyed more than half of the 400-year-old building.\
 """)
+
+
+async def test_anthropic_extended_thinking_with_tool_use(allow_model_requests: None, anthropic_api_key: str):
```

Comment:
With the suggested change to raise an error instead, we can modify this to try just one `output_type` and then use `pytest.raises` to verify the error